escpos/impl/epson.py

```diff
@@ -285,6 +286,12 @@ def cut(self, partial=True):
         """
         Trigger cutter to perform partial (default) or full paper cut.
         """
+
+        # Norberto Hideaki Enomoto -> Python 3.4
+
+        param = '\x01' if partial else '\x00'
+        self.device.write('\x1D\x56' + param)
+
```

Comment:
This does not make any sense. You just send cut command inconditionaly to device, whether or not the device has a cutter feature, triggering a double cut if cutter feature is enabled.